### DISCUSSION TITLE
Datasets_End Example Question

### DISCUSSION THREAD

**Discussion by dbenner63** [ibois-epfl/compas_wood#6] [Q&A]
Hey,

Trying to figure out how you got your data tree to sparse the information in pairs like in the image below.
![image](https://user-images.githubusercontent.com/75349064/227463201-1912cc7c-3d33-4b0b-9ab1-50980e2b8f79.png)

When i try something similar i've gotten it down to the 2 sets of poly lines but the tree is not like this at all. See below.
![image](https://user-images.githubusercontent.com/75349064/227463811-0d7f7159-6b09-47b0-a359-62afe88e25cc.png)

Please help!

**Comment by petrasvestartas** [comment]
Can you upload the grasshopper code here?

You can partition into two big lists, and then use swap matrix.
